test/fixtures/test-runner/output/force_exit.js

```diff
@@ -0,0 +1,25 @@
+// Flags: --test-force-exit --test-reporter=spec
+'use strict';
+const { after, afterEach, before, beforeEach, test } = require('node:test');
+
+before(() => {
+  console.log('BEFORE');
+});
+
+beforeEach(() => {
+  console.log('BEFORE EACH');
+});
+
+after(() => {
+  console.log('AFTER');
+});
+
+afterEach(() => {
+  console.log('AFTER EACH');
+});
+
+test('passes but oops', () => {
+  setInterval(() => {}, 5000);
```

Comment:
Another option: set timeout and then throw an error. This way, if force exit is broken the test can fail sooner, before hitting a timeout